src/services/MiniCryptoManagerImpl.js

```diff
@@ -0,0 +1,55 @@
+const { HDNode, ECSignature } = require('bitcoinjs-lib');
+
+/**
+ * MiniCryptoManagerImpl - A minimal CryptoManagerImpl for the portable CryptoManagerInterface
+ * to provide only default sign() and verify() functions to credential-commons with minimal dependencies.
+ *
+ * Particularities of this Mini Implementation:
+ *
+ *  1. Only sign() and verify() function are available;
+ *
+ *  2. This implementation is based on HDNode key material from bitcoinjs library;
+ *
+ *  3. There is no Storage Implementation nor any key storage as well. All "keyName" parameters are treated as
+ *    the own key in a base58 format ready to be used for sign or verify;
+ */
+class MiniCryptoManagerImpl {
+  /**
+   * Return input data signed using the specified key.
+   *
+   * Signature return value will be to be a DER encoded value.
+   *
+   * @param { string } keyName - The pvtKey in base58 format to be used to sign.
+   *                             NOTE: This is not just a keyName!
```

Comment:
If this is not just the keyName doesn’t the naming itself is invalid?

Reply:
@jpsantosbh me and @saviogrossi can't reach consensus on this, can you review please?

Reply:
I was just trying to follow the CryptoManager interface definition there.